.surge/index.html

```diff
@@ -187,15 +187,13 @@ <h2 class="card__heading">React CMF</h2>
 				</div>
 
 				<div class="card">
-					<a href="components" class="card__link">
-						<h2 class="card__heading">Components</h2>
-						<p class="card__body">
-							A set of stateless components which follows the Talend Guidelines.
-						</p>
+					<a href="design-system" class="card__link">
+						<h2 class="card__heading">Design System</h2>
+						<p class="card__body">Next gen components</p>
```

Comment:
```suggestion
						<h2 class="card__heading">Coral</h2>
						<p class="card__body">Talend Design System</p>
```

Reply:
```suggestion
						<h2 class="card__heading">Coral</h2>
						<p class="card__body">Talend's Design system documentation and React library</p>
```
To be as explicit as we can